packages/grafana-data/src/utils/csv.ts

```diff
@@ -205,6 +206,13 @@ function writeValue(value: any, config: CSVConfig): string {
 }
 
 function makeFieldWriter(field: Field, config: CSVConfig): FieldWriter {
+  if (field.display) {
+    return (value: any) => {
+      const displayValue = field.display!(value);
+      return writeValue(formattedValueToString(displayValue), config);
+    };
+  }
+
```

Comment:
The if statements below for `boolean` and `number` could probably be removed too? WDYT?

Reply:
On second thought, they should stay when we don't use field overrides setting CSV.